README.md

```diff
@@ -430,6 +430,46 @@ public function getUser($credentials, UserProviderInterface $userProvider)
 The logged-in user will be an instance of `KnpU\OAuth2ClientBundle\Security\User\OAuthUser` and will
 have the roles `ROLE_USER` and `ROLE_OAUTH_USER`.
 
+## Storing and refreshing tokens
+
+You have a couple of options to store access tokens for use at a later time:
+
+1. Store the `AccessToken` object (eg. serializing into the session), this allows you to check expiry before refreshing
+    ```php
+    // Fetch and store the AccessToken
+    $accessToken = $client->getAccessToken();
+    $session->set('access_token', $accessToken);
+
+    // Load the access token from the session, and refresh if required
+    $accessToken = $session->get('access_token');
+
+    if ($accessToken->hasExpired()) {
+        $accessToken = $client->refreshAccessToken($accessToken);
```

Comment:
needs to be `$accessToken->getRefreshToken()`, right?

Reply:
Good catch!